EssentialFeed/EssentialFeediOS/ImageComments UI/Views/ImageCommentCell.swift

```diff
@@ -0,0 +1,8 @@
+
+import UIKit
+
+public final class ImageCommentCell: UITableViewCell {
+	@IBOutlet private(set) public var labelUsername: UILabel!
+	@IBOutlet private(set) public var labelDate: UILabel!
+	@IBOutlet private(set) public var labelMessage: UILabel!
```

Comment:
The convention is to add the view type at the end of the name:

```suggestion
	@IBOutlet private(set) public var usernameLabel: UILabel!
	@IBOutlet private(set) public var dateLabel: UILabel!
	@IBOutlet private(set) public var messageLabel: UILabel!
```